code/datums/gamemode/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -281,7 +281,7 @@
 	role_category = /datum/role/wizard
 	my_fac = /datum/faction/wizard
 	enemy_jobs = list("Security Officer","Detective","Head of Security", "Captain")
-	required_enemies = list(2,2,1,1,1,1,1,0,0,0)
+	required_pop = list(20,20,15,15,15,15,15,10,10,0)
```

Comment:
I'm more baffled that an individual wizard requires less pop than ragin mages

Reply:
roundstart wiz has the summon gunk spells.